app/locale/en.coffee

```diff
@@ -702,9 +710,9 @@
     more_info_3: "is a good place to connect with fellow educators who are using CodeCombat."
 
   teachers_quote:
```

Comment:
I'm leaving the internal naming in place for now since we may change the terminology again in the near future and it doesn't seem too confusing as is.